tests/test_unix.py

```diff
@@ -362,6 +363,7 @@ def connection_lost(self, exc):
         self.assertIn(excs[0].__class__,
                       (BrokenPipeError, ConnectionResetError))
 
+    @unittest.skipUnless(sys.version_info < (3, 7), 'Python version must be < 3.7')
```

Comment:
If you have some light about why this testing is failing with `3.7` please tell me.